resources/assets/js/annotations/ol/PolygonValidator.js

```diff
@@ -0,0 +1,142 @@
+import JstsLinearRing from 'jsts/org/locationtech/jts/geom/LinearRing';
+import JstsPolygon from 'jsts/org/locationtech/jts/geom/Polygon';
+import LinearRing from '@biigle/ol/geom/LinearRing';
+import LineString from '@biigle/ol/geom/LineString';
+import Monkey from 'jsts/org/locationtech/jts/monkey'; // Needed for isValid(), normalize()
```

Comment:
This can be converted to a [side effect import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import?retiredLocale=de#forms_of_import_declarations). Then JSLint won't complain anymore.